pallets/schemas/src/lib.rs

```diff
@@ -278,9 +285,14 @@ pub mod pallet {
 			model: BoundedVec<u8, T::SchemaModelMaxBytesBoundedVecLimit>,
 			model_type: ModelType,
 			payload_location: PayloadLocation,
+			grants: BoundedVec<Grant, T::SchemaModelMaxBytesBoundedVecLimit>,
```

Comment:
Question: Do we really need a new extrinsic, since grants are optional, this Option should prevent any breaking changes if anyone is actually consuming this ?

Reply:
I think for backwards compatibility we will need a new extrinsic since the signature of the extrinsic in metadata will change